### PR TITLE
implement simple http handler

### PR DESCRIPTION
**Pull request by ayeniblessing101**
- Implement simple HTTP Handler
- Moved the server and models package to the internal directory because it will be used by the application

### CODE REVIEW

File: internal/server/server.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	"log"
+	"net/http"
+)
+
+func handleHelloWorld(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "%s", "Hello Dima")
+}
+
+// Server method handles all requests
+func Server() {
```
Comment (dkotik) [inline internal/server/server.go]: Always try to think of parameters. You should never have a function with no parameters other than the main() function. Minimally, it should return an error (that could be nil, meaning no error). In this case a listenTo string will also work nicely.

```
func Server(listenTo string) error {

}
```

Comment (dkotik) [reply internal/server/server.go]: Rarely a function will not return any error, when you are dealing with very simply string and number operations - but even in such cases other parameters are present.

**Review by dkotik — APPROVED** [review]
Good work, now study this package: https://github.com/davecgh/go-spew
I use it with many projects to examine complicated structs, for logging, debugging templates, etc. See if you can dump a &Recipe{} into the HTTP connection using spew.Sdump()

File: internal/server/server.go (hunk 1 above)
Comment (ayeniblessing101) [reply internal/server/server.go]: Ok... thanks for the feedback. If I get correctly We are making the port dynamic by passing in the parameter listenTo right?

**Pull request merged**